### ISSUE TITLE
Misleading comment (SVG filter)

### ISSUE THREAD

**Issue by mooffie** [Modernizr/Modernizr#2418]
`svg/filters.js` contains the comment "Should fail in Safari". This gives the impression that the test fails to correctly detect  the availability of SVG filters in Safari. This impression is wrong.

The detection works correctly for Safari.

I tested it on two Safaris:

Safari 5.1: the test returns `false`.
Safari 9.1: the test returns `true`.

This concurs with https://caniuse.com/#feat=svg-filters (which says the feature is in Safari 6+).

In short, I think the comment should be removed.

**Comment by rejas** [comment]
Sounds reasonable @mooffie Would you like to submit a PR for this?
